terra-workspace-manager/sa.tf

```diff
@@ -49,13 +55,25 @@ resource "google_project_iam_member" "app" {
   role     = local.app_sa_roles[count.index]
   member   = "serviceAccount:${google_service_account.app[0].email}"
 }
+// TODO(PF-156): Remove this once WM uses RBS
 resource "google_folder_iam_member" "app" {
   count    = local.create_folder ? length(local.app_folder_roles) : 0
   provider = google.target
   folder   = google_folder.workspace_project_folder[0].id
   role     = local.app_folder_roles[count.index]
   member   = "serviceAccount:${google_service_account.app[0].email}"
 }
+# Grant WorkspaceManager Service App Service Account permission to modify resource in folder.
+resource "google_folder_iam_member" "app_folder_roles" {
+  // Skip if google_folder variable is not present.
+  count = var.enable ? length(local.folder_ids_and_roles): 0
```

Comment:
per other PR, now I think we should try to make this `local.folder_ids_and_roles` a set of objects and `for_each` instead of count index. But we don't have to and can use `count` for consistency.
https://www.terraform.io/docs/configuration/meta-arguments/count.html#when-to-use-for_each-instead-of-count

Reply:
Done